test/Microsoft.ML.Tests/TrainerEstimators/SymSgdClassificationTests.cs

```diff
@@ -20,5 +26,42 @@ public void TestEstimatorSymSgdClassificationTrainer()
             TestEstimatorCore(pipe, dataView);
             Done();
         }
+
+        [Fact]
+        public void TestEstimatorSymSgdInitPredictor()
+        {
+            (var pipe, var dataView) = GetBinaryClassificationPipeline();
+            var transformedData = pipe.Fit(dataView).Transform(dataView);
+
+            // No actual training happens. 
+            var args = new LinearClassificationTrainer.Arguments();
+            var initPredictor = new LinearClassificationTrainer(Env, args, "Features", "Label").Fit(transformedData);
+            var data = initPredictor.Transform(transformedData);
+
+            var withInitPredictor = new SymSgdClassificationTrainer(Env, "Features", "Label").Train(transformedData, initialPredictor: initPredictor.Model);
+            var outInitData = withInitPredictor.Transform(transformedData);
+
+            var notInitPredictor = new SymSgdClassificationTrainer(Env, "Features", "Label").Train(transformedData);
+            var outNoInitData = notInitPredictor.Transform(transformedData);
+
+            int numExamples = 10;
+            var col1 = data.GetColumn<float>(Env, "Score").Take(numExamples).ToArray();
+            var col2 = outInitData.GetColumn<float>(Env, "Score").Take(numExamples).ToArray();
+            var col3 = outNoInitData.GetColumn<float>(Env, "Score").Take(numExamples).ToArray();
+
+            bool col12Diff = default;
+            bool col23Diff = default;
+            bool col13Diff = default;
+
+            for (int i = 0; i < numExamples; i++)
+            {
+                Console.WriteLine(col1[i].ToString() + " " + col2[i].ToString() + " " + col3[i].ToString());
```

Comment:
> Console.WriteLine(col1[i [](start = 15, length = 25)

do you need it in console, or file?

Reply:
Sorry that was just meant for debugging, I will remove that.